File: personal_expenses/android/app/src/main/kotlin/managertest/dimitarivanov/com/personal_expenses/Transaction.kt
package managertest.dimitarivanov.com.personal_expenses

import io.realm.RealmObject
import io.realm.annotations.PrimaryKey

//import io.realm.annotations.RealmClass

//@RealmClass
open class Transaction(
        @PrimaryKey
        open var uuid: String? = null,
        open var title: String? = null,
        open var amount: Double? = null,
        open var date: String? = null,
        open var orgName: String? = null,
        open var psAddress: String? = null,
        open var psName: String? = null
) : RealmObject()

File: personal_expenses/ios/Runner/Transaction.swift
//
//  Transaction.swift
//  Runner
//
//  Created by Dimitar Ivanov on 7.11.19.
//  Copyright © 2019 The Chromium Authors. All rights reserved.
//
import Realm

class Transaction : RLMObject {
    // Optional string property, defaulting to nil
    @objc dynamic var uuid = ""
    @objc dynamic var title = ""
    @objc dynamic var amount = 0.0
    @objc dynamic var date = ""
    
    override class func primaryKey() -> String? {
        return "uuid"
    }
}
